### ISSUE TITLE
compute and networking packages should be merged into one

### ISSUE THREAD

**Issue by martinsawicki** [Azure/azure-sdk-for-java#950]
compute already has a strong dependency on networking
in networking, to provide a user-friendly experience for load balancers, we should enable strongly typed references to VMs (and eventually likely VM scale sets). 
All the several alternative approaches we have considered have issues. 
The affinity between networking and compute is very high. It is hard to imagine a scenario where a user would only want networking and absolutely object to compute API coming along.
Some of the other major cloud providers have networking and compute in the same package as well.

**Comment by martinsawicki** [comment]
@jianghaolu, would you be able to take care of this?  I'm blocked by this now with further progress on LB

**Comment by jianghaolu** [comment]
I wonder what's the best way to get people's opinions on this?

**Comment by martinsawicki** [comment]
One way would be to do it and see what happens… :-)

**Comment by martinsawicki** [comment]
ok, maybe we don't have to do this after all...
